metrics/xrootd_metrics.go

```diff
@@ -280,17 +282,17 @@ var (
 	TransferReadvSegs = promauto.NewCounterVec(prometheus.CounterOpts{
 		Name: "xrootd_transfer_readv_segments_count",
 		Help: "Number of segments in readv operations",
-	}, []string{"path", "ap", "dn", "role", "org", "proj"})
+	}, []string{"path", "ap", "dn", "role", "org", "proj", "network"})
```

Comment:
Any reason we want to name it `network` instead of something like `ip`?

Reply:
@bbockelm Had mentioned using the `network` label in #1445, 
> given we are using networks and not hostnames, let's rename the labels to "network" perhaps?  Looking around on Google, it's not clear if there's a standard practice here.